embedJson/embed_json.go

```diff
@@ -0,0 +1,8 @@
+package embedJson
```

Comment:
Why did we pull this into its own package instead of just importing and assigning `embed.FS` in `cli/init`, since that's the only file where it is used?

Reply:
Because of how embedding files works, the embedded variable needs to be in a parent directory of the files that we want to embed, which is why I renamed the parent directory that we store the JSON files in as well.